cmd/observe/observe.go

```diff
@@ -95,9 +95,13 @@ application level. Rich filtering enable observing specific flows related to
 individual pods, services, TCP connections, DNS queries, HTTP requests and
 more.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := handleArgs(ofilter, vp.GetBool(config.KeyDebug)); err != nil {
+			debug := vp.GetBool(config.KeyDebug)
+			if err := handleArgs(ofilter, debug); err != nil {
 				return err
 			}
+			if debug {
+				fmt.Fprintf(cmd.ErrOrStderr(), "Using filters:\n=> whitelist: %s\n=> blacklist: %s\n", ofilter.whitelist, ofilter.blacklist)
```

Comment:
```suggestion
				fmt.Fprintf(cmd.ErrOrStderr(), "Using filters:\n=> include: %s\n=> exclude: %s\n", ofilter.whitelist, ofilter.blacklist)
```

nit: While the API field itself is called `blacklist/whitelist` and therefore cannot easily be renamed, we have been trying to move away from the blacklist/whitelist terminology.